client/pkg/fileutil/fileutil.go

```diff
@@ -135,3 +136,24 @@ func CheckDirPermission(dir string, perm os.FileMode) error {
 	}
 	return nil
 }
+
+// RemoveTempFile deletes useless temporary file with prefix tmp on an existing dir
+// Returns error if the dir does not exist or remove file fail
+func RemoveTempFile(dir string) error {
+	if !Exist(dir) {
+		return fmt.Errorf("directory %q does not exist.", dir)
+	}
+	fileNames, err := ReadDir(dir)
+	if err != nil {
+		return err
+	}
+	for _, fileName := range fileNames {
+		if strings.HasPrefix(fileName, "tmp") {
+			file := filepath.Join(dir, fileName)
+			if err = os.Remove(file); err != nil {
+				return err
```

Comment:
Let's not exit on first error, but:
  - try to delete all the files, 
  - Optionally: log all the errors
  - return error if there was problem with deletion of at least a single file.

I wan't to avoid situation where lack of permission on a single 'tmp' file is blocking cleanup alltogether.

Reply:
Done